chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -158,11 +159,14 @@
               kernel: "discovery/#{arch}/vmlinuz0")
   end
 
-  bash "Copy UEFI shim loader with grub2" do
-    cwd "#{uefi_dir}/default/boot"
-    code shim_code
-    action :nothing
-    subscribes :run, resources("template[#{uefi_dir}/default/grub.cfg]"), :immediately
+  # Secure Boot Shim
+  if arch == "x86_64"
+    bash "Copy UEFI shim loader with grub2" do
+      cwd "#{uefi_dir}/default/boot"
+      code shim_code
+      action :nothing
+      subscribes :run, resources("template[#{grubcfgfile}]"), :immediately
```

Comment:
Don't you have to update the template resource call as well to make it match the new path?